prometheus_es_exporter/scheduler.py

```diff
@@ -1,21 +1,27 @@
 import time
 import logging
+import concurrent.futures
```

Comment:
Doesn't look like this is used? If you could drop this as well it'd be great. Thanks!